scripts/log-search

```diff
@@ -90,7 +90,7 @@ NGINX_LOG_LINE_RE = re.compile(
          (?P<date> \d+/\w+/\d+ )
          :
          (?P<time> \d+:\d+:\d+ )
-         \s+ \+0000
+         \s+ [+-]\d+
```

Comment:
We should probably try to switch servers that are not UTC to UTC for consistency, but fixing this makes sense in any case.